crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -56,6 +56,9 @@ const MAX_SEND_BUF_BYTES: usize = 16000;
 /// gossiped with if gossip is triggered.
 const MAX_TRIGGERS: u8 = 2;
 
+/// The timeout for a gossip round if there is no contact. Five minutes.
+const ROUND_TIMEOUT_MS: u32 = 1000 * 60 * 5;
```

Comment:
It would be great if we used durations for consts from now on so we don't run into the unit conversion issues we had before, especially now that we're migrating to microseconds. Duration implements `Ord` so there's no problem for timeout checks.

```suggestion
const ROUND_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(60 * 5);
```